lib/widgets/nutrition/charts.dart

```diff
@@ -476,192 +584,3 @@ class MealDiaryBarChartWidgetState extends State<MealDiaryBarChartWidget> {
     );
   }
 }
-
-class FlNutritionalDiaryChartWidget extends StatefulWidget {
```

Comment:
Wouldn't this historical view be nice to have?

Reply:
I guess it could be useful somewhere, somehow. It's not very practical though. (Various UX issues, see discord). And I don't we should have both the chart *and* the table on the same page